stream-chat-android-ui-components-sample/src/main/kotlin/io/getstream/chat/ui/sample/feature/component_browser/messagepreview/ComponentBrowserMessagePreviewFragment.kt

```diff
@@ -29,15 +33,89 @@ class ComponentBrowserMessagePreviewFragment : Fragment() {
         _binding = null
     }
 
+    @InternalStreamChatApi
```

Comment:
???

Reply:
Uses `ChannelInfo` in its body, this makes that compile (it propagates the opt-in requirement, but since it's a framework method, it doesn't matter). Didn't want to add any opt-in Gradle configuration to this module.